src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public abstract class Task {
+    private final String taskName;
+    private boolean status; // True = Done, False = Not Done
```

Comment:
could perhaps change status to isCompleted so the code may be more understandable!